proto/generate.sh

```diff
@@ -67,8 +67,9 @@ $DOCKER_RUN /bin/bash -c 'set -x && JS_PULUMIRPC=/nodejs/proto && \
     mkdir -p "$TEMP_DIR"                       && \
     protoc --js_out=$JS_PROTOFLAGS:$TEMP_DIR --grpc_out=grpc_js,minimum_node_version=6:$TEMP_DIR --plugin=protoc-gen-grpc=/usr/bin/grpc_tools_node_protoc_plugin $PROTO_FILES && \
     find $TEMP_DIR && \
-    sed -i "s|^var global = .*;|var proto = { pulumirpc: {} }, global = proto;|" "$TEMP_DIR"/**/*.js && \
-    sed -i "s|require('\''../pulumi/|require('\''./|" "$TEMP_DIR"/pulumi/*.js && \
+    find $TEMP_DIR -type f -name "*.js" -exec sed -i "s|^var global = .*;|var proto = { pulumirpc: { codegen: { }, testing: { } }, google: { rpc: { } } }, global = proto;|" {} \; && \
```

Comment:
This is overkill, but its probably simpler than having a separate sed command for each directory.